ngo_design.css

```diff
@@ -0,0 +1,1315 @@
+@media (min-width: 480px) and (max-width: 800px){
+    body{
+        margin:0px;
```

Comment:
Space after :. 
https://www.learnhowtoprogram.com/intro-to-programming-evening/git-html-css/html-indentation-and-spacing
Pay attention to the last  paragraph of the above article

Reply:
given indentation to the css file